easybuild/easyblocks/s/siesta.py

```diff
@@ -0,0 +1,162 @@
+                         'Util/Contrib/APostnikov/vib2xsf', 'Util/Contrib/APostnikov/fmpdos'])
+
+            bins.extend(['Util/Denchar/Examples/2dplot.py', 'Util/Denchar/Examples/surf.py',
+                         'Util/Denchar/Src/denchar'])
+
+            bins.extend(['Util/DensityMatrix/dm2cdf', 'Util/DensityMatrix/cdf2dm'])
+
+            bins.extend(['Util/Eig2DOS/Eig2DOS'])
+
+            bins.extend(['Util/Gen-basis/ionplot.sh', 'Util/Gen-basis/gen-basis', 'Util/Gen-basis/ioncat'])
+
+            bins.extend(['Util/Grid/grid2cdf', 'Util/Grid/cdf2xsf', 'Util/Grid/cdf2grid', 'Util/Grid/grid2val',
+                         'Util/Grid/grid2cube'])
+
+            bins.extend(['Util/HSX/hsx2hs', 'Util/HSX/hs2hsx'])
+
+            bins.extend(['Util/Helpers/get_chem_labels'])
+
+            bins.extend(['Util/MPI_test/pi3'])
+
+            bins.extend(['Util/Macroave/Src/macroave'])
+
+            bins.extend(['Util/ON/lwf2cdf'])
+
+            bins.extend(['Util/Optimizer/swarm', 'Util/Optimizer/simplex'])
+
+            bins.extend(['Util/Projections/orbmol_proj'])
+
+            bins.extend(['Util/STM/simple-stm/plstm'])
+
+            bins.extend(['Util/SiestaSubroutine/FmixMD/Src/simple', 'Util/SiestaSubroutine/FmixMD/Src/driver',
+                         'Util/SiestaSubroutine/FmixMD/Src/para'])
+
+            bins.extend(['Util/TBTrans/tbtrans'])
+
+            bins.extend(['Util/VCA/mixps', 'Util/VCA/fractional'])
+
+            bins.extend(['Util/Vibra/Src/fcbuild', 'Util/Vibra/Src/vibrator'])
+
+            bins.extend(['Util/WFS/readwf', 'Util/WFS/readwfx', 'Util/WFS/info_wfsx', 'Util/WFS/wfs2wfsx',
+                         'Util/WFS/wfsx2wfs', 'Util/WFS/wfsnc2wfsx'])
+
+            bins.extend(['Util/pdosxml/pdosxml'])
+
+            bins.extend(['Util/pseudo-xml/xml2psf'])
+
+            bins.extend(['Util/test-xml/test-xml'])
+
+        self.cfg['files_to_copy'] = [(bins, 'bin')]
+
+    def sanity_check_step(self):
+        """Custom sanity check for Siesta."""
+
+        bins = ['bin/siesta']
+
+        if self.cfg['with_transiesta']:
+            bins.extend(['bin/transiesta'])
+
+        if self.cfg['with_utils']:
+            bins.extend(['bin/' + util for util in 'eigfat2plot', 'new.gnubands', 'gnubands', 'ccViz', 'mprop',
+                         'dm_creator', 'fat', 'eig2bxsf', 'xv2xsf', 'md2axsf', 'rho2xsf', 'vib2xsf', 'fmpdos',
+                         '2dplot.py', 'surf.py', 'denchar', 'dm2cdf', 'cdf2dm', 'Eig2DOS', 'ionplot.sh', 'gen-basis',
+                         'ioncat', 'grid2cdf', 'cdf2xsf', 'cdf2grid', 'grid2val', 'grid2cube', 'hsx2hs', 'hs2hsx',
+                         'get_chem_labels', 'pi3', 'macroave', 'lwf2cdf', 'swarm', 'simplex', 'orbmol_proj', 'plstm',
+                         'simple', 'driver', 'para', 'tbtrans', 'mixps', 'fractional', 'fcbuild', 'vibrator', 'readwf',
+                         'readwfx', 'info_wfsx', 'wfs2wfsx', 'wfsx2wfs', 'wfsnc2wfsx', 'pdosxml', 'xml2psf', 'test-xml'
```

Comment:
High, I suppose...

Just saw that they are changing their license and development platform (http://departments.icmab.es/leem/siesta/News/siesta-GPL.txt), but the build system is the same.

Looked at siesta-4.0-b2, at least gnubands is not longer built, only new.gnubands, so at least than can be dropped, will check more differences.

We could of course make all of this simpler and more generic if we build in install dir and use `ConfigureMake` instead of `MakeCp`, keeping all the utils in their original place, which I guess is what the developers intend...